gbcamextract.c

```diff
@@ -38,6 +38,7 @@ const int FILE_ERROR = 2;
 const int FILE_SIZE_ERROR = 3;
 
 const int WIDTH = 160;
+const int ROW_SIZE = WIDTH / 4; // 2 bits per pixel means 4 pixels per byte
```

Comment:
Even though WIDTH is const, it is not _constant_, and so it cannot be used in the initializer here. I commit a change that sets ROW_SIZE to a literal 40. These should probably be #defines but who cares.